File: C++/ejercicio1.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

int main() {

    // Creamos el vector para guardar los nombres (equivalente a ArrayList)
    vector<string> nombres;

    // Agregamos 5 nombres de estudiantes
    nombres.push_back("Ana");
    nombres.push_back("Luis");
    nombres.push_back("Maria");
    nombres.push_back("Carlos");
    nombres.push_back("Sofia");

    // Mostramos todos los nombres con un ciclo for
    cout << "Lista de estudiantes:" << endl;
    for (int i = 0; i < nombres.size(); i++) {
        cout << (i + 1) << ". " << nombres[i] << endl;
    }

    return 0;
}

File: Java/Ejercicio1.java
import java.util.ArrayList;

public class Ejercicio1 {
    public static void main(String[] args) {

        // Creamos el ArrayList para guardar los nombres
        ArrayList<String> nombres = new ArrayList<>();

        // Agregamos 5 nombres de estudiantes
        nombres.add("Ana");
        nombres.add("Luis");
        nombres.add("Maria");
        nombres.add("Carlos");
        nombres.add("Sofia");

        // Mostramos todos los nombres con un ciclo for
        System.out.println("Lista de estudiantes:");
        for (int i = 0; i < nombres.size(); i++) {
            System.out.println((i + 1) + ". " + nombres.get(i));
        }
    }
}